### ISSUE TITLE
Should connection pooling be in the Sails documentation?

### ISSUE THREAD

**Issue by arnthorsnaer** [balderdashy/sails-docs#546]
I belive it should.

Last December, @mikermcneil added this message to sails-postgresql/lib/adapter.js :

> To override the default `poolSize`, specify a `poolSize` property on the relevant Postgresql,  "connection" config object. If you\'re using Sails, this is generally located in `config/connections.js`, or wherever your environment-specific database configuration is set.

Sails-postgresql does not mention poolSize in the readme, nor did I find it in the sails-postgresql code, so I wonder if this is a sails property or specific to the postgresql adapter. I did find this in the sails-mongodb default options but that is besides the point. The main point is that I find no mention of poolSize or connection pooling at all in the sails. 

From various sources I have found mention of the following settings:
* pool  
* poolSize
* connectionLimit 
* waitForConnections   

Since the sails documentation on this is so sparse, it is difficult for me to know which of the properties are adapter specific and which properties can be used regardless of adapters. Given that there is semantic overlap in poolSize and connectionLimit, I could *guess* that these two properties are adapter specific. As mentioned before I found no mention of this in the sails-postgresql readme.

Here are the two docs pages where I would expect this to be covered:
* http://sailsjs.org/documentation/reference/configuration/sails-config-connections
* http://sailsjs.org/documentation/anatomy/my-app/config/connections-js

**Comment by sailsbot** [comment]
Thanks for posting, @arnthorsnaer.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by JHereu** [comment]
I know this post is from Aug 2015, but is there any news about it, such as some minimum documentation already?

**Comment by qnk** [comment]
I've not found any documentation about it and I think it is quite necessary.
